homeassistant/components/pooldose/const.py

```diff
@@ -0,0 +1,38 @@
+"""Constants for the Seko Pooldose integration."""
+
+from __future__ import annotations
+
+from homeassistant.helpers.device_registry import CONNECTION_NETWORK_MAC, DeviceInfo
+
+DOMAIN = "pooldose"
+DEFAULT_HOST = "KOMMSPOT"
+MANUFACTURER = "SEKO"
+
+
+def device_info(info: dict | None, unique_id: str) -> DeviceInfo:
+    """Return device info dict for Pooldose device."""
+    info = info or {}
+    api_version = info.get("API_VERSION")
+    # Remove trailing slash from API version
+    if api_version:
+        api_version = api_version[:-1]
+    return DeviceInfo(
```

Comment:
we should leave const.py for constants by the way, so the best place for this is entity.py